workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/camunda/bpmn/builder/FormRepliedNodeBuilder.java

```diff
@@ -29,15 +29,10 @@ public class FormRepliedNodeBuilder extends AbstractNodeBpmnBuilder {
   public AbstractFlowNodeBuilder<?, ?> build(WorkflowNode element, String parentId,
       AbstractFlowNodeBuilder<?, ?> builder, BuildProcessContext context) {
     if (builder instanceof ParallelGatewayBuilder) {
-      return builder.exclusiveGateway()
-          .intermediateCatchEvent()
-          .timerWithDuration(readTimeout(element))
```

Comment:
we don't put timeout duration ?

Reply:
we can have a short discuss on it. i think it is not necessary